### DISCUSSION TITLE
Updated nvim-metals - having trouble connecting to build server

### DISCUSSION THREAD

**Discussion by tylerheaton16** [scalameta/nvim-metals#457] [General]
Hey guys, I've been using nvim-metals for awhile now and am seeing an odd error where the semanticdb.jar file is being referenced in a cache directory which isn't a space I own (another colleague). I'm not quiet sure where this path is being picked up. 

Is there a search order for bloop that is somehow finding this before it gets to my cached location?

![image](https://user-images.githubusercontent.com/25470591/190280511-a1aaf580-6b72-4096-be80-7fb6dce7d0e4.png)

**Comment by ckipp01** [comment]
Hey @tylerheaton16. So typically this type of thing can happen if someone has accidentally committed their `.bloop` directory. For example if you would try in a fresh hello-world workspace I'm assuming this would work fine. Could you try that to verify?

**Reply by tylerheaton16** [reply]
I noticed my java version was 17.04. I downgraded to 11.0 and these errors went away.
